components/notebook-controller/config/crds/notebook_v1alpha1_notebook.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
so kustomize would be using these files? Are they used now as part of deployment of the controller?